package org.purpleBean.kmip.codec.ttlv.deserializer.kmip.common;

import org.purpleBean.kmip.codec.ttlv.deserializer.AbstractKmipTtlvDeserializer;
import org.purpleBean.kmip.common.KeyPartIdentifier;

public class KeyPartIdentifierTtlvDeserializer extends AbstractKmipTtlvDeserializer<KeyPartIdentifier, Integer> {

    public KeyPartIdentifierTtlvDeserializer() {
        super(KeyPartIdentifier.kmipTag, KeyPartIdentifier.encodingType, Integer.class, value -> KeyPartIdentifier.builder().value(value).build());
    }
}